jobs/nats/spec

```diff
@@ -47,11 +47,13 @@ properties:
   nats.cluster_port:
     description: "The port for the NATS servers to communicate with other servers in the cluster."
     default: 4223
+  nats.tls_cluster_port:
+    description: "The port for the NATS servers to communicate with other servers in the cluster (no default but usually 4225)."
```

Comment:
nit: can you remove the parens and make this description match the other one you added

> "The port for the NATS servers to communicate with other servers in the cluster. No default but usually 4223."